app/assets/javascripts/the_bridge_ng/controllers/marketplace_ctrl.js.coffee

```diff
@@ -26,9 +27,25 @@
       false
 
   filtersMatch = (product, filters) ->
+    
     if filters.integrated
       return product.integrated
     else
       return true
+
+  $scope.changeVote = (product) ->
+  
+    
+    $http.get("/products/#{product.id}/vote").success((data) ->
+      console.log(data)
+      return product.demo = data
```

Comment:
good work guys, I'm not sure what it means by `.demo`.

The payload from `#vote` should be namespace by `product`

    product = data.product